compiler/qsc_parse/src/item.rs

```diff
@@ -561,12 +562,31 @@ fn parse_import_or_export(s: &mut ParserContext) -> Result<ImportOrExportDecl> {
 
 fn parse_import_or_export_item(s: &mut ParserContext) -> Result<ImportOrExportItem> {
     // an import item is a path followed by an optional alias,
-    let path = *(path(s)?);
+    let mut idents = Vec::default();
+    let mut is_glob = false;
+    loop {
+        idents.push(*ident(s)?);
+        if s.peek().kind == TokenKind::Dot {
+            s.advance();
+        }
```

Comment:
So what if the next token isn't a dot?

The below parses as an import item currently :)

```qsharp
import Foo Bar Baz *;
```

You may want to use a loop like `while token(s, TokenKind::Dot).is_ok()`